### ISSUE TITLE
[Feature] Read-only mode should have the option to be rendered on the server for SEO.💡

### ISSUE THREAD

**Issue by ranemihir** [codex-team/editor.js#1355]
The newly introduced Read-only mode can be used to put the JSON data directly into the main Editor instance to display the data as an article. But right now the main Editor instance is rendered on the client side which does not give SEO.

My idea is to provide some way to render the main Editor instance on the server side but we have to write universal code for this (code which works on both client and server side which does not use client specific objects like document).

**Comment by bcnroller** [comment]
Not being able to satisfy SEO requirements is a show-stopper for me. The editor must provide a solution that also does not rely on React or Angular.

**Comment by Duncank** [comment]
I disagree, since the content that has been entered into the editor can be used in many different ways it doesn't make sense to render the content 'as is', as each users implementation will vary.
You are able to create your own rendered view based on the editor's data, aren't you?

**Comment by SuperWallaby** [comment]
It's a hack method, but what if you render the contents to a dummy Div and replace that part with Editor readOnly?
